Azure.ResourceManager.Core/AzureResourceManagerClient.cs

```diff
@@ -164,17 +164,17 @@ public IEnumerable<Location> ListAvailableLocations(string subscription, Resourc
                 .Locations.Cast<Location>();
         }
 
-        public ResourceGroupOperations ResourceGroup(string subscription, string resourceGroup)
+        public ResourceGroupOperations GetResourceGroupOperations(string subscription, string resourceGroup)
```

Comment:
params should be more explicit
```suggestion
        public ResourceGroupOperations GetResourceGroupOperations(string subscriptionGuid, string resourceGroupName)
```